<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>592</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>739</width>
    <height>592</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>739</width>
    <height>592</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(56, 56, 56);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="sidebar" native="true">
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>180</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(44, 30, 124);</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>../../Downloads/favicon_io/android-chrome-192x192.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_5" native="true">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>250</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>250</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="btn_home">
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>140</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>JetBrainsMono Nerd Font</family>
              <pointsize>13</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">.QPushButton{
color: rgb(255, 255, 255);
border-radius: 10px;
}
.QPushButton:hover{
border: 2px solid #38aecc;
border-radius: 10px;
}</string>
            </property>
            <property name="text">
             <string>Home</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../../Downloads/favicon_io/favicon-16x16.png</normaloff>../../Downloads/favicon_io/favicon-16x16.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_history">
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>140</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>JetBrainsMono Nerd Font</family>
              <pointsize>13</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">.QPushButton{
color: rgb(255, 255, 255);
border-radius: 10px;
}
.QPushButton:hover{
border: 2px solid #38aecc;
border-radius: 10px;
}</string>
            </property>
            <property name="text">
             <string>History</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../../Downloads/favicon_io/favicon-16x16.png</normaloff>../../Downloads/favicon_io/favicon-16x16.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>140</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>JetBrainsMono Nerd Font</family>
              <pointsize>13</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">.QPushButton{
color: rgb(255, 255, 255);
text-align: center;
border-radius: 10px;
}
.QPushButton:hover{
border: 2px solid #38aecc;
border-radius: 10px;
}</string>
            </property>
            <property name="text">
             <string>Log</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../../Downloads/favicon_io/favicon-16x16.png</normaloff>../../Downloads/favicon_io/favicon-16x16.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_4">
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>140</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>JetBrainsMono Nerd Font</family>
              <pointsize>13</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">.QPushButton{
color: rgb(255, 255, 255);
border-radius: 10px;
}
.QPushButton:hover{
border: 2px solid #38aecc;
border-radius: 10px;
}</string>
            </property>
            <property name="text">
             <string>About</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../../Downloads/favicon_io/favicon-16x16.png</normaloff>../../Downloads/favicon_io/favicon-16x16.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_6" native="true">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>50</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_23">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>35</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Segoe Print</family>
              <pointsize>11</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color:rgb(255, 255, 255)</string>
            </property>
            <property name="text">
             <string>Powered By</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_24">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Segoe MDL2 Assets</family>
              <pointsize>19</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>Secware</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignBottom|Qt::AlignHCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="content" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="home_page">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QFrame" name="frame">
             <property name="font">
              <font>
               <family>JetBrainsMono Nerd Font Propo</family>
               <pointsize>16</pointsize>
              </font>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>60</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>60</height>
                 </size>
                </property>
                <property name="baseSize">
                 <size>
                  <width>0</width>
                  <height>300</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>JetBrainsMono Nerd Font Mono</family>
                  <pointsize>28</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 255, 255);</string>
                </property>
                <property name="text">
                 <string>Home</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignHCenter|Qt::AlignTop</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QFrame" name="frame_2">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>80</height>
                 </size>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QLineEdit" name="edit_watch_folder">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>40</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>JetBrainsMono Nerd Font</family>
                     <pointsize>12</pointsize>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">color: rgb(255, 255, 255);</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="btn_file_chooser">
                   <property name="minimumSize">
                    <size>
                     <width>80</width>
                     <height>40</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>JetBrainsMono Nerd Font</family>
                     <pointsize>12</pointsize>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="cursor">
                    <cursorShape>PointingHandCursor</cursorShape>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">QPushButton{color: rgb(44, 30, 124);
background-color: rgb(55, 174, 204);
border-radius: 10px;
}</string>
                   </property>
                   <property name="text">
                    <string>Select</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QFrame" name="frame_3">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>80</height>
                 </size>
                </property>
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <widget class="QPushButton" name="btn_start">
                   <property name="minimumSize">
                    <size>
                     <width>80</width>
                     <height>40</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>JetBrainsMono Nerd Font</family>
                     <pointsize>12</pointsize>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="cursor">
                    <cursorShape>PointingHandCursor</cursorShape>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">QPushButton{color: rgb(44, 30, 124);
	color: rgb(255, 255, 255);
background-color: rgb(85, 170, 0);
border-radius: 10px;
}</string>
                   </property>
                   <property name="text">
                    <string>Start Watching</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="btn_stop">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>80</width>
                     <height>40</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>JetBrainsMono Nerd Font</family>
                     <pointsize>12</pointsize>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="cursor">
                    <cursorShape>PointingHandCursor</cursorShape>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">QPushButton{color: rgb(44, 30, 124);
background-color: rgb(185, 0, 0);
	color: rgb(255, 255, 255);
border-radius: 10px;
}</string>
                   </property>
                   <property name="text">
                    <string>Stop Watching</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item alignment="Qt::AlignHCenter">
               <widget class="QLabel" name="label_watch">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>60</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>JetBrainsMono Nerd Font</family>
                  <pointsize>18</pointsize>
                  <italic>true</italic>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(85, 255, 255);</string>
                </property>
                <property name="text">
                 <string>Stoped</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="history_page">
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>150</x>
             <y>160</y>
             <width>201</width>
             <height>161</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Histroy Label</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
